jumpscale/packages/vdc_dashboard/chats/monitoring_stack.py

```diff
@@ -2,36 +2,68 @@
 from jumpscale.sals.chatflows.chatflows import chatflow_step
 from jumpscale.sals.vdc import VDCFACTORY
 from jumpscale.packages.vdc_dashboard.sals.solutions_chatflow import SolutionsChatflowDeploy
+from jumpscale.sals.marketplace import deployer
```

Comment:
I don't think we should use this deployer 
https://github.com/threefoldtech/js-sdk/blob/ac9efc8570eff5c14954b928f225cf84f8945c3c/jumpscale/packages/vdc_dashboard/sals/solutions_chatflow.py#L28